src/nvim/buffer.c

```diff
@@ -5372,7 +5383,8 @@ bool bt_nofile(const buf_T *const buf)
                          || buf->b_p_bt[0] == 'p');
 }
 
-// Return true if "buf" is a "nowrite", "nofile" or "terminal" buffer.
+// Return TRUE if "buf" is a "nowrite", "nofile", "terminal" or "prompt"
```

Comment:
Same issue here with changing `true` to `TRUE` even though `bt_dontwrite` is a boolean function.